lending-service/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM eclipse-temurin:17-jdk-jammy AS build
+WORKDIR /workspace
+
+COPY gradlew gradlew.bat settings.gradle ./
+COPY gradle gradle
+COPY common common
+COPY lending-service lending-service
+
+RUN chmod +x gradlew
+RUN ./gradlew :lending-service:bootJar --no-daemon
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)